src/WebJobs.Script/Workers/Rpc/Configuration/LanguageWorkerOptionsSetup.cs

```diff
@@ -42,18 +47,53 @@ public void Configure(LanguageWorkerOptions options)
         {
             string workerRuntime = _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
 
-            // Parsing worker.config.json should always be done in case of multi language worker
+            // Parsing worker.latestConfiguration.json should always be done in case of multi language worker
```

Comment:
Is `worker.latestConfiguration.json` a new thing?

Reply:
This is looking like an accidental change to me. I don't expect a different file name.